### PR TITLE
Update URL for attribute 'config'

### PR DESCRIPTION
**Pull request by dbarcosb**
Relative URLs are disallowed and currently it does not validate. Changing it to absolute URL

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Can you please accept the CLA

File: examples/source/1.components/amp-call-tracking.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
     We also need to set a default phone number that will be shown before the endpoint responds. This is done via an `<a>` element inside the `amp-call-tracking` tag.
   -->
-    <amp-call-tracking config="/static/samples/json/call-tracking-number.json">
+    <amp-call-tracking config="https://amp.dev/static/samples/json/call-tracking-number.json">
```
Comment (matthiasrohmer) [inline examples/source/1.components/amp-call-tracking.html]: Actually

```suggestion
    <amp-call-tracking config="<% hosts.platform %>/static/samples/json/call-tracking-number.json">
```

would be nice to be environment agnostic 🙂

Comment (sebastianbenz) [reply examples/source/1.components/amp-call-tracking.html]: good point!

**Comment by dbarcosb** [comment]
Done, signed!

**Comment by sebastianbenz** [comment]
@dbarcosb could you please accept @matthiasrohmer's suggestion.

**Pull request merged**